music_assistant/server/providers/snapcast/__init__.py

```diff
@@ -133,6 +134,20 @@ async def get_config_entries(
             hidden=not local_snapserver_present,
             help_link="https://raw.githubusercontent.com/badaix/snapcast/86cd4b2b63e750a72e0dfe6a46d47caf01426c8d/server/etc/snapserver.conf",
         ),
+        ConfigEntry(
+            key=CONF_SERVER_CHUNK_MS,
+            type=ConfigEntryType.INTEGER,
+            range=(10, 100),
+            default_value=20,
+            label="Snapserver chunk size",
+            description="Default source stream read chunk size [ms]. "
+            "The server will continously read this number of milliseconds from the source into buffer and pass this buffer to the encoder."
+            "The encoded buffer is sent to the clients. Some codecs have a higher latency and will need more data, e.g. Flac will need ~26ms chunks",
+            required=False,
+            category="Built-in Snapserver Settings",
+            hidden=not local_snapserver_present,
+            help_link="https://raw.githubusercontent.com/badaix/snapcast/86cd4b2b63e750a72e0dfe6a46d47caf01426c8d/server/etc/snapserver.conf",
+        ),
```

Comment:
How does the chunk_size work in the snapserver? isn't it defined by default different for each codec? i don't know why i thought it was that way but now i can't find anything in the documentation

Reply:
To be honest, I am not sure how exactly this parameter is working either. I just know from my experience that if this value is too small the sound will sometimes become choppy. Due to this I think it would be handy to leave out a configurable parameter in case of problems.